src/Components/LandingPage/LandingPage.js

```diff
@@ -18,15 +20,33 @@ import {
   PageHeader,
   PageHeaderTitle,
 } from '@redhat-cloud-services/frontend-components';
+import { useChrome } from '@redhat-cloud-services/frontend-components/useChrome';
 import { Outlet } from 'react-router-dom';
 
+import awsQuickStart from './awsquickstart.json';
+import azureQuickStart from './azurequickstart.json';
+import contentQuickStart from './contentquickstart.json';
+import './LandingPage.scss';
+
 import isBeta from '../../Utilities/isBeta';
 import ImagesTable from '../ImagesTable/ImagesTable';
-import './LandingPage.scss';
 import DocumentationButton from '../sharedComponents/DocumentationButton';
 
 export const LandingPage = () => {
   const [showBetaAlert, setShowBetaAlert] = useState(true);
+  const [showHint, setShowHint] = useState(true);
+
+  const { quickStarts } = useChrome();
+  const activateQuickstart = (qs) => quickStarts.toggle(qs.metadata.name);
```

Comment:
Is there a reason why we're not calling on the quickstarts currently defined on the platform? Duplicating the content feels not great.

Reply:
AFAIK those are available only on stage. But let me check.

Reply:
no still not there

Reply:
It is fine if it's just in stage, Martin said he can push it within minutes. That should not be a reason to copy content

Reply:
I spoke with him on Tuesday and he mentioned it's blocked by QE at the moment. 

@Hyperkid123 Hey Martin, is there an ETA when we can switch to using `quickStarts.activateQuickstart()`?